pkg/vault/vault.go

```diff
@@ -230,6 +230,7 @@ func GetSecret(secretEngine string, path string, keys []string, version int) (ma
 			return nil, ErrParseSecret
 		}
 	}
+	log.Info("GetSecret: Get data from vault", "secretData is ", secretData)
```

Comment:
Can you remove this log line or at least the `secretData`. I think it can be problematic if the secrets are visible in the logs. For example we are using Elasticsearch where more people have access to as to Vault, therefor they shouldn't have access to the secrets.